### ISSUE TITLE
Console only shows first time

### ISSUE THREAD

**Issue by bladecoder** [StrongJoshua/libgdx-inGameConsole#27]
Hi,

I updated from 0.5.3 to 0.7.0 and now the console only shows the first time I push the assigned key.

In my case I have set the key <F1> with console.setDisplayKeyID(Keys.F1);

Steps to reproduce it:

1. Push F1 -> the console shows
2. Push F1 -> the console hides
3. Push F1 -> the console **doesn't** show anymore.

Best Regards.

**Comment by aitor3ml** [comment]
same thing happens to me

**Comment by aitor3ml** [comment]
Forcing the visible toggle will do the trick until we get an answer.
```
@Override
public boolean keyDown(int keyCode) {
    if (keyCode == console.getDisplayKeyID()) {
        console.setVisible(!console.isVisible());
        return true;
    }
```

**Comment by StrongJoshua** [comment]
I am currently without a computer, but I will investigate this as soon as I am able.

**Comment by StrongJoshua** [comment]
This should be fixed in 0.7.1
Please let me know of any other issues should they arise!
